### PR TITLE
Sils/eslint

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/700/commits/d30cc4f394ab31657f44526fc2a118b9a23c2c4e

File: bears/js/ESLintBear.py [hunk 1]
```diff
@@ -31,15 +31,23 @@ class ESLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         eslint_config: str=""):
+                         eslint_config: str=''):
```
Comment (hypothesist) [inline bears/js/ESLintBear.py]: Unrelated change

File: bears/js/ESLintBear.py [hunk 2]
```diff
@@ -31,15 +31,23 @@ class ESLintBear:
...
         """
         :param eslint_config: The location of the .eslintrc config file.
         """
         args = '--no-ignore', '--no-color', '-f=json', '--stdin'
+        if not eslint_config:
+            args += ('--config', config_file)
+
         if eslint_config:
             args += ('--config', eslint_config)
```
Comment (hypothesist) [inline bears/js/ESLintBear.py]: @sils1297 are you sure there won't be an issue because of two `--config` flags? They'll be appended internall?

Comment (hypothesist) [inline bears/js/ESLintBear.py]: Ah never mind, that's not possible

File: tests/js/ESLintBearTest.py [hunk 3]
```diff
@@ -31,13 +31,14 @@
                             "eslintconfig.json")
 
 ESLintBearTestWithConfig = verify_local_bear(ESLintBear,
-                                             valid_files=(
-                                                 test_good, ''),
-                                             invalid_files=(test_bad,),
+                                             valid_files=('',),
+                                             invalid_files=(test_bad,
+                                                            test_good),
```
Comment (hypothesist) [inline tests/js/ESLintBearTest.py]: misleading variable name then

File: tests/js/ESLintBearTest.py [hunk 4]
```diff
@@ -31,13 +31,14 @@
...
                                              settings={"eslint_config":
                                                        eslintconfig})
 
 ESLintBearWithoutConfig = verify_local_bear(ESLintBear,
                                             valid_files=(
-                                                test_good, test_bad, ''),
-                                            invalid_files=(test_syntax_error,))
+                                                test_good, ''),
```
Comment (hypothesist) [inline tests/js/ESLintBearTest.py]: Oh you've used it elsewhere too. Never mind

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/700/commits/87b974d369b0ca63d3b597204ba11e11ab1be70d

**Comment by sils1297** [comment]
@rultor merge
